.github/workflows/sanity-check.yml

```diff
@@ -1,5 +1,12 @@
 name: Sanity check
-on: [pull_request]
+on: 
+  pull_request:
+    types:
+    - "opened"
+    - "reopened"
+    - "synchronize"
+    - "labeled"
+    - "unlabeled"
```

Comment:
What's the difference between the old and the new here? Are there some types that it used to run on but now won't?